azure-pipelines.yml

```diff
@@ -1341,6 +1278,169 @@ jobs:
       includeRootFolder: false
       sevenZipCompression: 5
       archiveFile: $(Build.ArtifactStagingDirectory)/zip/lpk-linux-arm64.zip
+  
+  - task: PublishBuildArtifacts@1
+    displayName: Generate lks.json for .zip files
+    inputs:
+      PathtoPublish: $(Build.ArtifactStagingDirectory)/zipv2
+      ArtifactName: zipv2
+
+  - task: SdtReport@2
```

Comment:
These same tasks are added later on in job `Phase_3` (lines 1415-1428). I'm not sure what they do, so I can't tell whether they need to be in both places. Should they be?

Reply:
I have removed these tasks, initially I thought this should work but then realized these tasks won't be available in mac os runners. Thanks for pointing this out.

Reply:
They are not duplicate, they are guardian security analysis reporting which is required for SDL process and keeping the pipeline and generated binaries secured.